src/main/java/org/mitre/caasd/commons/out/GzFileSink.java

```diff
@@ -158,6 +158,7 @@ public class GzFileSink<T> implements Consumer<T>, Closeable {
      *                       are generated you'll open too many file handles and kill performance or
      *                       just crash.
      * @param expirationTime How long between writing pieces of data until a .gz file is closed
+     * @throws Gibberish
```

Comment:
😆

Reply:
oof.  I hope this got removed later.  It was add to check if a javadoc screwup made the build fail.  (which was true at some point)